config/environments/production.rb

```diff
@@ -8,6 +8,15 @@
     }
   }
 
+  config.cache_store = :dalli_store,
+  (ENV["MEMCACHIER_SERVERS"] || "").split(","),
+  {:username => ENV["MEMCACHIER_USERNAME"],
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.